samples/interactive-tutorials/src/test/java/search/SearchSimpleQueryTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package search;
+
+import com.google.cloud.retail.v2.SearchResponse;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import util.StreamGobbler;
+
+public class SearchSimpleQueryTest {
+  private String output;
+  private String defaultSearchPlacementName;
+
+  @Before
+  public void setUp() throws IOException, InterruptedException, ExecutionException {
+    String projectNumber = System.getenv("PROJECT_NUMBER");
+    String defaultCatalogName =
+        String.format("projects/%s/locations/global/catalogs/default_catalog", projectNumber);
+    defaultSearchPlacementName = defaultCatalogName + "/placements/default_search";
+    Process exec =
+        Runtime.getRuntime()
+            .exec("mvn compile exec:java -Dexec.mainClass=search.SearchSimpleQuery");
+    StreamGobbler streamGobbler = new StreamGobbler(exec.getInputStream());
+    Future<String> stringFuture = Executors.newSingleThreadExecutor().submit(streamGobbler);
+    output = stringFuture.get();
```

Comment:
What's going on here?

Reply:
If you're just trying to get the output, I suggest you try this method: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/kms/src/test/java/kms/SnippetsIT.java#L108-L116